bot/resources/tags/customcooldown.md

```diff
@@ -0,0 +1,20 @@
+**Cooldowns**
+
+Cooldowns can be used in discord.py to rate-limit. In this example, we're using it in an on_message.
+
+```python
+from discord.ext import commands
+
+message_cooldown = commands.CooldownMapping.from_cooldown(1.0, 60.0, commands.BucketType.user)
+
+@bot.event
+async def on_message(message):
+    bucket = message_cooldown.get_bucket(message)
+    retry_after = bucket.update_rate_limit()
```

Comment:
I think it's worth mentioning what this return value is. It's hard to tell what "retry after" means if one hasn't encountered that term before.